crates/editor/src/display_map.rs

```diff
@@ -97,7 +97,7 @@ pub struct DisplayMap {
     /// Decides where the fold indicators should be and tracks parts of a source file that are currently folded.
     fold_map: FoldMap,
     /// Keeps track of hard tabs in a buffer.
-    tab_map: TabMap,
+    tab_map: InvisibleMap,
```

Comment:
We should rename these other references too.

Reply:
let me change those thanks...